CHANGES.rst

```diff
@@ -32,6 +32,10 @@ In development
   previous versions of Joblib.
   https://github.com/joblib/joblib/pull/1374
 
+- Add a ``return_generator`` parameter for ``Parallel``, that allows
+  to consume results asynchronously.
+  https://github.com/joblib/joblib/pull/1393
+
```

Comment:
I think we should also document the simplification of the traceback for the sequential case.